.gitignore

```diff
@@ -10,14 +10,12 @@ _CPack_Packages/
 Testing
 Makefile
 cmake_install.cmake
+cmake-build-debug/
```

Comment:
I suggest making this `cmake-build-*` because the suffix changes with the release mode and toolchain (when you have more than one defined)